### PR TITLE
Add metadata for DOC OCR 22.4

### PR DESCRIPTION
**Pull request by irinel-birsan-uipath**

### CODE REVIEW

File: metadata/UiPathDocumentOCR_CPU__2__metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "name": "UiPathDocumentOCR_CPU",
+  "outputDescription": "Please refer to the documentation of the Activity used to query the ML Skill.",
+  "processorType": "CPU",
+  "projectId": "[project-id]",
+  "retrainable": false,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Document Understanding",
+  "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "v21.10.1",
```
Comment (vnaren23) [inline metadata/UiPathDocumentOCR_CPU__2__metadata.json:20]: I think this should be 22.4

File: metadata/UiPathDocumentOCR__7__metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "name": "UiPathDocumentOCR",
+  "outputDescription": "Please refer to the documentation of the Activity used to query the ML Skill.",
+  "processorType": "GPU",
+  "projectId": "[project-id]",
+  "retrainable": false,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Document Understanding",
+  "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "v21.10.1",
```
Comment (vnaren23) [inline metadata/UiPathDocumentOCR__7__metadata.json:20]: Same as above.

File: metadata/UiPathDocumentOCR_CPU__2__metadata.json (hunk 1 above)
Comment (irinel-birsan-uipath) [reply metadata/UiPathDocumentOCR_CPU__2__metadata.json]: Done

File: metadata/UiPathDocumentOCR__7__metadata.json (hunk 2 above)
Comment (irinel-birsan-uipath) [reply metadata/UiPathDocumentOCR__7__metadata.json]: Done

**Review by mohitseth-nitrr — APPROVED** [review]
LGTM

**Pull request merged**